### PR TITLE
added cron job to comment ongoing referenda on PRs

### PR DESCRIPTION
**Pull request by Bullrich**
Added functionality for a cron job to iterate over all ongoing referendas and compare them to the open PRs. If there is any match on the remark, and the referenda has been opened **after** the last time the action was run, it will comment a link to the referenda in Polkassembly.

This is intended to resolve the requirements in polkadot-fellows/RFCs#57

### CODE REVIEW

File: src/rfc-cron.ts [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    } else {
+      console.log("Had an error while creating RFC for #%s", pr.number, rfcResult.error);
+    }
+  }
+
+  return prRemarks;
+};
+
+export const getAllRFCRemarks = async (startDate: Date): Promise<{ url: string; remark: string }[]> => {
+  const wsProvider = new WsProvider("wss://polkadot-collectives-rpc.polkadot.io");
```
Comment (rzadp) [inline src/rfc-cron.ts:58]: Remember about `process.env.PROVIDER_URL`

**Comment by mordamax** [comment]
@Bullrich is it only for new referendas? 
can it track the existing referendas status changes as it's requested in https://github.com/paritytech/eng-automation/issues/2

**Review by mordamax — APPROVED** [review]
Would love to see it working for a several days on stg before implementation
How you think the further referenda status changes will be tracked?

**Comment by Bullrich** [comment]
Hi @rzadp! It is ready for review.

I don't really know what tests I can add aside from ensuring that it doesn't crash when fetching the referenda. If you have any suggestions, please, let me know.

**Comment by mordamax** [comment]
> Do you know how to setup an alternative testing environment so I can start testing it?

answered in our matrix channel - could be Westend -> Collectives with making us fellows there installed to paritytech-stg?

> You mean when they close?

as described here https://github.com/paritytech/eng-automation/issues/2

File: README.md [hunk 2]
```diff
@@ -74,3 +74,33 @@ The built-in `secrets.GITHUB_TOKEN` can be used, as long as it has the necessary
...
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+jobs:
+  notify_referendas:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get last run
+        run: echo "last=$(gh run list -w "$WORKFLOW" --json startedAt -q '.[0].startedAt')" >> "$GITHUB_OUTPUT"
```
[the remark this answers is not in the document]
Comment (Bullrich) [reply README.md:95]: Good suggestion, I added it, but I skipped the `limit 1` as, if the last one failed, we don't want to take all existing ongoing referendas and post them again.

**Pull request merged**